test/EFCore.MySql.IntegrationTests/Tests/Models/GeneratedTypesTest.cs

```diff
@@ -11,7 +11,7 @@ namespace Pomelo.EntityFrameworkCore.MySql.IntegrationTests.Tests.Models
 {
 	public class GeneratedTypesTest
 	{
-		[SkipAppVeyorFact]
+		[SkipAppVeyorAndMariaDbFact]
```

Comment:
```
[SkipAppVeyorFact]
[SkipMariaDbFact]
```

Reply:
@caleblloyd xUnit2 doesn't allow multiple test attributes on a method. I wanted them seperate too :/
`Message: System.InvalidOperationException : Test method 'Pomelo.EntityFrameworkCore.MySql.IntegrationTests.Tests.Models.GeneratedTypesTest.TestGeneratedContact' has multiple [Fact]-derived attributes
`

Reply:
Ah I did not know.  Makes sense how you have it then